src/init/mod.rs

```diff
@@ -6,6 +6,7 @@ pub mod wait;
 
 mod clickhouse_exporter;
 mod datadog_exporter;
+pub mod file_exporter;
```

Comment:
Can this remain non public?

Reply:
This one is tricky.

Making file_exporter non-public breaks access to its public types from outside init -- specifically `FileExporterFormat`. The other exporter modules are private because their types are not used outside init.